task_schedule_check_cron.cfg

```diff
@@ -17,19 +17,14 @@ heartbeat    task_sched_heartbeat_check_cron
 
 # Email addresses that receive an alert if there was a severe error in running jobs.
 
-alert       aca@cfa.harvard.edu
+alert       taldcroft@cfa.harvard.edu
 
 # Define task parameters
-
+# Apparently check_cron cannot be the only thing in the job so we make a dummy exec.
```

Comment:
I assume the original design is that watch_cron_logs would be called directly for that case?  But it has been a while.